src/utils.ts

```diff
@@ -295,3 +295,15 @@ export const getSuccessRateIcon = (
     return 'times-square' as IconName;
   }
 };
+
+export function getRandomProbes(probes: number[], quantity: number): number[] {
+  if (quantity >= probes.length) {
+    return probes;
+  }
+  const randomProbes = new Set([] as number[]);
+  while (randomProbes.size < quantity) {
+    const index = Math.floor(Math.random() * probes.length);
+    randomProbes.add(probes[index]);
+  }
+  return Array.from(randomProbes).sort();
```

Comment:
What kind of sorting are you expecting here? I assume it's not a real sort. So why call sort then?

Reply:
It's an array of integers, so I'm just expecting the default